src/fluent-api/playwright-fluent.ts

```diff
@@ -1833,11 +1833,30 @@ export class PlaywrightFluent implements PromiseLike<void> {
       this.actions.push(async (): Promise<void> => await (story as Story)(this));
       return this;
     }
-

```

Comment:
Hi @osolomin90 , there is still a subtle change, could you revert the suppression of line 1836 ?

Reply:
sure